pkg/apiserver/experiment/experiment.go

```diff
@@ -63,3 +69,184 @@ func (s *Service) getExperimentDetail(c *gin.Context) {}
 
 // TODO: need to be implemented
 func (s *Service) deleteExperiment(c *gin.Context) {}
+
+// getPodChaosState returns the state of PodChaos
+func (s *Service) getPodChaosState (stateInfo map[string]int) error {
+	var chaosList v1alpha1.PodChaosList
+	err := s.kubeCli.List(context.Background(), &chaosList)
+	if err != nil {
+		return err
+	}
+	for _, chaos := range chaosList.Items {
+		stateInfo["Total"]++
+		switch chaos.Status.ChaosStatus.Experiment.Phase {
+		case v1alpha1.ExperimentPhaseRunning:
+			stateInfo["running"]++
```

Comment:
s/runnig/Running?

Reply:
And did we need to define Running = "Running"?

Reply:
Did we need a state like "New"?

Reply:
It has been modified and there is currently no "New" phase.